FONTLOG.txt

```diff
@@ -34,6 +34,16 @@ http://github.com/alif-type/xits
 
 ChangeLog
 ----------
+[Insert Date etc.] <XITS> Version 1.300
+- Add small caps ’smcp’ for the regular text font.
+- Add T and S with comma below for setting Romanian.
+- Add superior and inferior figures.
+- Add alternate old-style figures, accessible via ‘ss11’.
+- Fix misplaced accents on ‘i’.
+- Add ’ffj’ ligature.
+- Fix thickness of connector part in division symbol.
```

Comment:
I thought I fixed that one already.

Reply:
You did but it wasn't in fontlog as it was since the last release. So I added it to the notes for the forthcoming one